coco/request_forwarder.py

```diff
@@ -58,15 +59,22 @@ async def trigger(self, method, request=None, hosts=None, params=None):
         if not hosts:
             hosts = self.group
         forward_result = await self.forward_function(
-            self.name, method, request, hosts, params
+            self.name,
+            request,
+            hosts=hosts,
+            method=method,
+            params=params,
+            timeout=self.timeout,
         )
         if self.check:
             for check in self.check:
                 forward_result.success &= await check.run(forward_result)
 
         return forward_result
 
-    def forward_function(self, name, method, request, hosts=None, params=None):
+    def forward_function(
```

Comment:
Why did method get moved?

Reply:
So it can be ignored (`**_`) in the implementation for internal forwards.